pkg/BUILD.bazel

```diff
@@ -3376,45 +3376,59 @@ test_suite(
 )
 
 test_suite(
-    name = "small_tests",
+    name = "ccl_tests",
```

Comment:
Why not maintain the small, medium, large, enormous categories for CCL tests?

Reply:
Good question. Because we don't use them, we can easily add them if we need to though.